tools/testsys-config/src/lib.rs

```diff
@@ -0,0 +1,129 @@
+use serde::{Deserialize, Serialize};
+use std::fs;
+use std::path::{Path, PathBuf};
+
+/// Configuration needed to run tests
+#[derive(Debug, Default, Deserialize, Serialize, PartialEq, Eq)]
+#[serde(deny_unknown_fields)]
+pub struct TestConfig {
+    // High level configuration for TestSys
+    pub test: Option<Test>,
+}
+
+impl TestConfig {
+    /// Deserializes an TestConfig from a given path
```

Comment:
Nit:

```suggestion
    /// Deserializes a TestConfig from a given path
```

Same for all occurrences